packages/plugin-connector/src/data-sources/build-connector.js

```diff
@@ -9,7 +9,7 @@ import { DatabaseConnectorSchema } from './schemas/query-runner.schema';
 export const buildConnector = async (packageMain, supports) => {
 	// https://github.com/nodejs/node/issues/31710 thanks windows
 	const crossPlatformPackage = new URL(`file:///${packageMain}`).href;
-	const connectorPackage = await import(crossPlatformPackage);
```

Comment:
What changed to make this necessary?

Reply:
Vite complains in a few cases when importing the plugin connector because it can't optimize the dynamic dynamic import; so the ignore comment suppresses that warning